File: A's/cpp/2126A.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int t; cin >> t;
    while (t--) {
        int x; cin >> x;
        int y = 0;
        string x_s = to_string(x);
        string y_s = to_string(y);
        bool found = false;
        while (!found) {
            for (char c : y_s) {
                if (x_s.find(c) != string::npos) {
                    cout << y << endl;
                    found = true;
                }
            }
            y++;
            y_s = to_string(y);
        }
    }
    
    return 0;
}

File: A's/py/2126A.py
def main() -> None:
    t: int = int(input())
    while t:
        x: int = int(input())
        y: int = 0
        found: bool = False
        while not found:
            for c in str(x):
                if c in str(y): 
                    print(y)
                    found = True
                    break
            y += 1
        t -= 1

if __name__ == "__main__":
    main()